zerver/lib/actions.py

```diff
@@ -3297,6 +3297,18 @@ def do_delete_message(user_profile, message):
     move_message_to_archive(message.id)
     send_event(event, ums)
 
+def do_update_attachment_list(user_profile, uri):
+    # type: (UserProfile, Any) -> None
+    path_id = uri.split("/")[2:]
+    path_id = '/'.join(path_id)
+    attachment = Attachment.objects.filter(path_id=path_id).first()
+    if attachment is not None:
+        attachment = attachment.to_dict()
+    event = {
+        'type': 'update_attachment_list',
```

Comment:
Make the the type "attachment", with op=add, for similarity to our other events.

You should also add changes in `zerver/lib/events.py` and `zerver/tests/test_events.py` for this.  

zulip.readthedocs.io/en/latest/new-feature-tutorial.html should give you the context you need to do that correctly.